### PR TITLE
ui. show ip address on multiwan active backup configuration

### PR DESCRIPTION
**Pull request by federicoballarini**
Refer to https://github.com/NethServer/dev/issues/5956

### CODE REVIEW

File: api/wan/read [hunk 1]
```diff
@@ -78,7 +78,11 @@ if($cmd eq 'providers') {
         $ret->{'configuration'}{'multiwan'}{$_} = $cdb->get_prop('firewall',$_);
     }
     $ret->{'configuration'}{'multiwan'}{'EmailAddress'} = $cdb->get_prop('root', 'EmailAddress');
-
+    if ($ret->{'configuration'}{'multiwan'}{'WanMode'} eq 'backup'){
```
Comment (gsanchietti) [inline api/wan/read]: Cosmetic comment on code indentation, add a space before '{`
```suggestion
    if ($ret->{'configuration'}{'multiwan'}{'WanMode'} eq 'backup') {
```

File: api/wan/read [hunk 2]
```diff
@@ -78,7 +78,11 @@ if($cmd eq 'providers') {
...
+		my $externalip = `dig +short myip.opendns.com \@resolver1.opendns.com`;
+		$ret->{'configuration'}{'multiwan'}{'CurrentIp'} = substr($externalip, 0, length($externalip)-1);
+	}
```
Comment (gsanchietti) [inline api/wan/read]: Cosmetic comment on code indentation, align the ` {` parenthesis to the parent `if`

File: ui/public/i18n/language.json [hunk 3]
```diff
@@ -157,7 +157,8 @@
         "in_out_bound_zero": "Upload or Download bandwidth is not set",
         "fireqos_temporary_disabled": "Traffic shaping is temporarily disabled during the speed test.",
         "speedtest_error": "Speedtest failed: please check Internet connection.",
-        "click_to_open": "Click to expand details"
+        "click_to_open": "Click to expand details",
+        "current_ip": "Current IP"
```
Comment (gsanchietti) [inline ui/public/i18n/language.json]:
```suggestion
        "current_ip": "Current public IP"
```

File: api/wan/read [hunk 4]
```diff
@@ -78,7 +78,11 @@ if($cmd eq 'providers') {
         $ret->{'configuration'}{'multiwan'}{$_} = $cdb->get_prop('firewall',$_);
     }
     $ret->{'configuration'}{'multiwan'}{'EmailAddress'} = $cdb->get_prop('root', 'EmailAddress');
-
+    if ($ret->{'configuration'}{'multiwan'}{'WanMode'} eq 'backup'){
+		my $externalip = `dig +short myip.opendns.com \@resolver1.opendns.com`;
+		$ret->{'configuration'}{'multiwan'}{'CurrentIp'} = substr($externalip, 0, length($externalip)-1);
```
Comment (gsanchietti) [inline api/wan/read]: If you're trying to remove the newline at the end of the output, just use:
```
chomp $externalip;
$ret->{'configuration'}{'multiwan'}{'CurrentIp'} = $externalip;
```

**Comment by gsanchietti** [comment]
@edospadoni this is how the changes looks&feel, it's good for me!
Screenshot from 2019-11-25 11-46-10

File: api/wan/read (hunk 1 above)
Comment (federicoballarini) [reply api/wan/read]: Changed.

File: api/wan/read (hunk 2 above)
Comment (federicoballarini) [reply api/wan/read]: Changed.

File: ui/public/i18n/language.json (hunk 3 above)
Comment (federicoballarini) [reply ui/public/i18n/language.json]: Changed.

File: api/wan/read (hunk 4 above)
Comment (federicoballarini) [reply api/wan/read]: Changed.

**Pull request merged**